### DISCUSSION TITLE
Boards and matrixes going out!

### DISCUSSION THREAD

**Discussion by davepl** [PlummersSoftwareLLC/NightDriverStrip#403] [Announcements]
I just programmed and packed up complete kits for the following folks:

Erich Smith, Sean Pearce, Kevin Craig, Bill Swearingen, Alfredo Rivera, Shawn Everingham, Kerian Hines, Brian B, Jordan Pinch, Paul Smith, and Andrew Wright.

If you want a board and don't see yourself on the list, email me confirmation of having built the source tree successfully, then email me with the title of BOARD REQUEST and include your mailing address.  

Priority goes to React developers at the moment!  But C++ contributions welcome too!

**Comment by stretch911** [comment]
Excited!

**Comment by SeanPGit** [comment]
Excellent! Can't wait!

Sean

**Comment by Unclerojelio** [comment]
What constitutes confirmation of having successfully built the source tree?

**Comment by davepl** [comment]
You’ll get a banner something like this!  It should say SUCCESS.

Archiving .pio/build/mesmerizer/libFrameworkArduino.a
Indexing .pio/build/mesmerizer/libFrameworkArduino.a
Linking .pio/build/mesmerizer/firmware.elf
Retrieving maximum program size .pio/build/mesmerizer/firmware.elf
Checking size .pio/build/mesmerizer/firmware.elf
Advanced Memory Usage is available via "PlatformIO Home > Project Inspect"
RAM:   [===       ]  26.0% (used 85176 bytes from 327680 bytes)
Flash: [=====     ]  52.6% (used 1724165 bytes from 3276800 bytes)
Building .pio/build/mesmerizer/firmware.bin
esptool.py v4.5.1
Creating esp32 image...
Merged 25 ELF sections
Successfully created esp32 image.
========================================================================= [SUCCESS] Took 27.58 seconds =========================================================================

Environment    Status    Duration
-------------  --------  ------------
mesmerizer     SUCCESS   00:00:27.575

**Reply by Unclerojelio** [reply]
Thanks!

**Comment by robertlipe** [comment]
[ unofficial answer. I just "work" here. ]

Probably something like a screen capture of the end of the successful build
and a directory listing of the generated .bin files or such.

Without reading minds or putting my words where they don't belong, my guess
is to show that you can successfully navigate the required tools,
processes, and directions to actually be productive and not
require hand-holding of the "what is a terminal?" variety.

Snobbish? Maybe. But an entry ticket isn't unreasonable since there's a
considerable investment being made on the other end...

I could be wrong.

**Reply by Unclerojelio** [reply]
No worries, I'll rebuild the project tonight and take a screen shot.

**Comment by SeanPGit** [comment]
@davepl Is there any tracking for the Boards packages? 
I'm excited and getting itchy feet seeing the fruits of others! 😂
With Brexit and all that - worried such stuff is going to get stuck in customs / attract charges (I don't mind!), and just wanted to track any progress! 🙃

**Reply by rbergen** [reply]
I'd indeed count on some delay due to customs handling and also be ready for VAT/customs charges. I'm in the Netherlands so basically "next door" to the UK, but since Brexit things I ordered from the UK have sometimes been stuck in customs for up to 3 weeks(!) and were then only delivered after paying VAT and handling fees. 

This shipment is obviously coming from the US, but I think the signs are that European customs organisations have generally been overloaded since Brexit came into effect.

**Comment by davepl** [comment]
I think I might have kept it, will check.  But I know they went out, because it cost me $50 a piece to ship the international ones!  Crazy!  Even Canada was expensive from here.

- Dave

**Reply by SeanPGit** [reply]
Hi Dave, complete radio silence here ... Fedex & Royal mail usually quite keen to "pass on their cost"... Got excited when I got a Fedex notification, then realised I'd ordered oil for my car! Happy to pay postage if you've held onto it for that reason!?

**Comment by davepl** [comment]
I’m happy to FedEx another one if you cover the cost, but it was $40 or so the first time, just be aware :-) But shoot me your address again if you like and I can get another one out to you!

Cheers,
Dave
